zcash_client_backend/src/data_api.rs

```diff
@@ -1229,14 +1225,19 @@ pub trait WalletRead {
         ufvk: &UnifiedFullViewingKey,
     ) -> Result<Option<Self::Account>, Self::Error>;
 
-    /// Returns the most recently generated unified address for the specified account, if the
-    /// account identifier specified refers to a valid account for this wallet.
+    /// Returns the most recently generated unified address for the specified account that conforms
+    /// to the specified address request, if the account identifier specified refers to a valid
+    /// account for this wallet.
+    ///
+    /// If the `request` parameter is `None`, this will be interpreted as a request for an address
+    /// having a receiver corresponding to each data item in the account's UIVK.
     ///
     /// This will return `Ok(None)` if the account identifier does not correspond to a known
-    /// account.
-    fn get_current_address(
+    /// account, or if no previously generated address conforms to the specified request.
```

Comment:
It seems undesirable to conflate those conditions. I think unknown account should always have been treated as an error here.

Reply:
Fixed in a89dd4389d2b2fd21c53b9f2c44ed920dc3bd609